File: src/ch05/ex05/DayChecker.java
package ch05.ex05;

import java.time.LocalDate;
import java.time.temporal.ChronoUnit;

public class DayChecker {

	private static final LocalDate BIRTHDAY = LocalDate.of(1986,  2, 18);

	public static void main(String[] args) {
		long days = BIRTHDAY.until(LocalDate.now(), ChronoUnit.DAYS);
		System.out.println("生きてきた日数：" + days);
	}
}

File: src/ch07/ex01/DayChecker.js
// DayChecker.java相当のことをJSでしてみる。

var BIRTHDAY = java.time.LocalDate.of(1986,  2, 18)

var days = BIRTHDAY.until(java.time.LocalDate.now(), java.time.temporal.ChronoUnit.DAYS)
print("生きてきた日数：" + days)

// Q. Javaでテストプログラムを書くより簡単か？
// A. main()を書かなくてよい分は簡単。パスを指定するのは面倒。